Easydict/NewApp/LanguagePreference/LanguageState.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  LanguageState.swift
+//  Easydict
+//
+//  Created by choykarl on 2024/3/3.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import SwiftUI
+
+// MARK: - LanguageState
+
+let kEZLanguagePreferenceLocalKey = "kEZLanguagePreferenceLocalKey"
```

Comment:
I think this constant string looks like objc style definition, maybe move to objc side or rename to like `LanguagePreferenceLocalKey`.